notes/hub/create-pull-request.md

```diff
@@ -0,0 +1,13 @@
+## Create Pull Request
+
+Use the following command to create a pull request from a commit:
+
+> hub pull-request -p --no-edit -o
```

Comment:
I tend to associate `>` with asides, less important information a user can skip over if they're not interested:

![image](https://user-images.githubusercontent.com/9750687/71785111-9ba12400-2fb0-11ea-9577-14d12bf44566.png)

Would prefer removing the `>` and wrapping in backticks (\`) since it's code